cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1122,7 +1122,7 @@ if(davix AND NOT builtin_davix)
     find_package(Davix 0.6.4)
     if(NOT DAVIX_FOUND)
       find_package(LibUUID)
-      if(NOT LibUUID_FOUND)
+      if(NOT TARGET LibUUID::LibUUID)
         message(STATUS "Davix dependency libuuid not found, switching OFF 'davix' option.")
```

Comment:
I'm wondering if we should error out here. Somebody seemed to have wanted davix, and it's surprising to not have enabled.

Reply:
probably when "fail on missing" in ON but then it should be at the level of DAVIX ... (and that failure seems to be not implemented ...)